packages/react-dom/src/shared/DOMProperty.js

```diff
@@ -206,6 +206,25 @@ export function getPropertyInfo(name: string): PropertyInfo | null {
   return properties.hasOwnProperty(name) ? properties[name] : null;
 }
 
+export function getCustomElementPropertyInfo(
+  name: string,
+  node: Element) {
+  if (name in (node: any)) {
+    const acceptsBooleans = (typeof (node: any)[name]) === 'boolean';
+    return {
+      acceptsBooleans,
+      type: acceptsBooleans ? BOOLEAN : STRING,
+      mustUseProperty: true,
+      propertyName: name,
+      attributeName: name,
+      attributeNamespace: null,
+      sanitizeURL: false,
```

Comment:
I think this is fine if we assume whatever it passes it to is going to sanitize it. However `<a href="javascript:..." is="my-custom-element">` or `<img src="..." is="my-custom-img" />` etc. needs to be sanitized since it might not be upgraded yet. That's an existing issue that we should fix.

Reply:
Oh my, I didn't even know about the `is` attribute. Does react treat elements with the `is` attribute like custom elements...? If not, then is this fine as is...?

Reply:
>Does react treat elements with the is attribute like custom elements...?

Yes.

Reply:
Ok, I don't think this should block this PR, but it would be good to follow up on later.
I'll make a note to do so.